apps/demo-app/src/app/app.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="docs-main">
   <mat-toolbar class="docs-header">
     <div class="doc-navigation">
-      <img class="logo header-element" src="../assets/images/entry_logo_small_transparent_material.png">
+      <img class="logo header-element" src="../assets/images/entry_logo_small_transparent_material.png" alt="Entry logo">
```

Comment:
You don't need to use relative paths for assets because they are already defined in the angular.json file

Reply:
good catch mate ;)